src/pages/SearchPage.js

```diff
@@ -231,9 +253,6 @@ export default compose(
         reports: {
             key: ONYXKEYS.COLLECTION.REPORT,
         },
-        personalDetails: {
-            key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-        },
```

Comment:
Okay I will look into this, thanks 👍

Reply:
@situchan I am trying to repro this but I can't. Can you tell me what shortcuts are you using or the repro steps if possible?


https://github.com/Expensify/App/assets/47336142/c0092f12-ac57-4b55-ab52-825718bfde6c

Reply:
Ah I tested on 2 devices. Let's say A has contact B!
- On A, search for B and keep search page open
- On B, update their display name (this should be done on another device)
- On A, B's profile is not updated on search page until close and re-open

I am reporting this because it doesn't happen on main

Reply:
@situchan I am testing now on main as you have suggested using two different accounts on web and android emulator. However, on main the result is the same, the personal details doesn't update in real-time and you have to switch to the user and get the latest data for personal details.

Can you show me in a video, how it works for you on the main using two different accounts?

### Main:
https://github.com/Expensify/App/assets/47336142/d26739e5-ce91-4ef7-b879-f7cc48d613c0

Reply:
I will test it one more time after clearing caches and verify Pusher is working.

Reply:
Okay so I was able to repro this and it's now fixed 👍